extension/obs/src/index.ts

```diff
@@ -246,28 +321,56 @@ class OBS extends EventEmitter {
         // OBS not enabled, don't even try to set.
         throw new Error('No connection available');
       }
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore: Typings say we need to specify more than we actually do.
-      await this.conn.send('SetSceneItemProperties', {
-        'scene-name': scene,
-        item: { name: item },
-        visible: visible ?? true,
-        position: {
-          x: area?.x ?? 0,
-          y: area?.y ?? 0,
+
+      // None of this is properly documented btw.
```

Comment:
I'm guessing this comment is about `callBatch` in particular? May be useful to add this note to the other use of it too if so.